src/abstract-view.js

```diff
@@ -9,6 +9,8 @@
 
 Marionette.AbstractView = Backbone.View.extend({
   isDestroyed: false,
+  _isShown: false,
```

Comment:
`CompositeView` uses _isRendered  
There's really not much reason these couldn't move into those two.
`CollectionView` has that '_onShowCalled` method and `CompositeView` has a `render` function.  I don't think there'd be need for the listeners.